platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ExportMigrationConfigurationAdminContext.java

```diff
@@ -105,13 +113,18 @@ private boolean isValid(Configuration cfg) {
     return ConfigurationAdminMigratable.isManagedServiceFactory(cfg);
   }
 
-  public Stream<ExportMigrationConfigurationAdminEntry> entries() {
-    return entries.values().stream();
+  public Stream<ExportMigrationEntry> fileEntries() {
```

Comment:
❓Don't we need to add unit tests for `fileEntries` and `memoryEntries`?

Reply:
Since the tests provided were higher level, I shall keep it like that. If I add validation for file entries as suggested in the previous comment, I think we should be good.